code/modules/mob/living/simple_animal/bot/hygienebot.dm

```diff
@@ -92,6 +93,8 @@
 	if(washing)
 		do_wash(loc)
 		for(var/AM in loc)
+			if (AM == src)
+				continue
```

Comment:
This feels like a lazy way to do it but I haven’t gotten around to fixing up hyienebot so Ill likely change it in the future.

Reply:
the simplest way is sometimes the best way